content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -727,6 +730,15 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `TopologyManager`: Enable a mechanism to coordinate fine-grained hardware resource
   assignments for different components in Kubernetes. See
   [Control Topology Management Policies on a node](/docs/tasks/administer-cluster/topology-manager/).
+- `TopologyManagerPolicyAlphaOptions`: Allow fine-tuning of topology manager policies,
+  experimental, Alpha-quality options.
+  This feature gate guards *a group* of topology manager options whose quality level is alpha.
+  This feature gate will never graduate to beta or stable.
+- `TopologyManagerPolicyBetaOptions`: Allow fine-tuning of topology manager policies,
+  experimental, Beta-quality options.
+  This feature gate guards *a group* of topology manager options whose quality level is alpha.
+  This feature gate will never graduate to stable.
```

Comment:
Same here. We don't want users to know what are behind the gate, and we want them to try them out?

Reply:
The specific options gated by this feature gate is described in the relevant documentation for the TopologyManager itself, e.g. 
`content/en/docs/tasks/administer-cluster/topology-manager.md`.

There is no precedent for listing out the specific options here (if that is what you are suggesting).